isomerpages-base/misc/custom.scss

```diff
@@ -4,16 +4,14 @@
 
 @charset "utf-8";
 
-// HLB brand colors
-$primary: #6031b6;
-$secondary: #4372d6;
+// Agency brand colors
+$primary: {{- site.colors.primary-color -}};
+$secondary: {{- site.colors.secondary-color -}};
 $secondary-hover: lighten($secondary, 10%);
 
-$media-color-one: #49759a;
-$media-color-two: #744d9f;
-$media-color-three: #00838f;
-$media-color-four: #00838f;
-$media-color-five: #00838f;
+{%- for media-color in site.colors.media-colors -%}
```

Comment:
We need to do this for lines 41 to 72 as well.

Reply:
done